src/workerd/jsg/README.md

```diff
@@ -1510,24 +1510,27 @@ als.run(123, () => scheduler.wait(10)).then(() => {
 console.log(als.getStore());  // undefined
 ```
 
-Any type can act as an async resource by acquiring a reference to the current Async
+Within C++, any type can act as an async resource by acquiring a reference to the current Async
 Context Frame.
 
 ```cpp
 jsg::Lock& js = ...;
 
-// enter the async resource scope:
+auto maybeAsyncContext = jsg::AsyncResource::current(js);
```

Comment:
Does this documentation need to be updated to refer to AsyncContextFrame rather than AsyncResource?

Can we avoid the word "resource" in this section? Despite its use in node, I find that terminology pretty confusing whereas "propagating the async context" seems straightforward.